epsagon/trace.py

```diff
@@ -569,6 +578,34 @@ def _strip(self):
 
             self.events_map.pop(event.identifier(), None)
 
+    @staticmethod
+    def _strip_key(key):
+        """
+        Strip a given key from spaces, dashes, and underscores.
+        :param key: The key to strip.
+        :return: Stripped key.
+        """
+        return key.lower().replace('-', '').replace('_', '').replace(' ', '')
+
+    def remove_ignored_keys(self, input_dict):
+        """
+        Remove ignored keys recursively.
+        :param input_dict: Input dict to remove ignored keys from.
+        :return: None
+        """
+        if not self.keys_to_ignore:
+            return
+        stripped_keys = [self._strip_key(x) for x in self.keys_to_ignore]
```

Comment:
I would run this line once at the init, rather than everytime we send trace

Reply:
Got it